memtable/inlineskiplist.h

```diff
@@ -279,7 +279,11 @@ struct InlineSkipList<Comparator>::Node {
   // next_[0].  This is used for passing data from AllocateKey to Insert.
   void StashHeight(const int height) {
     assert(sizeof(int) <= sizeof(next_[0]));
-    memcpy(&next_[0], &height, sizeof(int));
+    // Explicitly cast to void* because GCC 8 is (rightfully) upset that this is
+    // "writing to an object of type ‘...’ with no trivial copy-assignment".
```

Comment:
Our internal lint said, you have non-ascii chars (I assume "‘" and "’") in this line and you should fix it.

Reply:
Ah, copied verbatim from the gcc error message. Will fix tomorrow.

Reply:
Fixed.